decidim-proposals/config/locales/ca.yml

```diff
@@ -197,6 +197,7 @@ ca:
           proposal_in_evaluation_reason: S'està avaluant aquesta proposta
           proposal_rejected_reason: 'Aquesta proposta ha estat rebutjada perquè:'
           report: Informe
+          withdraw_btn_hint: Pots retirar la teva proposta si canvies d'opinió, sempre i quan no hagi rebut cap suport. La proposta no s'elimina, apareixerà a la llista de propostes retirades.
```

Comment:
These texts should be added via Crowdin, not manually.

Reply:
Ok. Only :en translation now, I'll translate :ca and :es via Crowdin when this PR is accepted 😄